extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -28,22 +32,34 @@ class AmpAccordion extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    /** @const @private {boolean} */
+    this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
+
     /** @const @private {!NodeList} */
     this.sections_ = this.getRealChildren();
 
     /** @const @private {!Window} */
     this.win_ = this.getWin();
 
-    /** @const @private {string} */
-    this.id_ = this.getSessionStorageKey_();
+    /** @const @private {string|null} */
+    this.sessionId_ = null;
+
+    /** @const @private {boolean} */
+    this.sessionOptOut_ = false;
+    if (this.isExperimentOn_) {
+      this.sessionOptOut_ = this.element.hasAttribute('store-state-opt-out');
```

Comment:
i am not convinced with this name. 
`disable-session-states`
`incognito` ?

@cramforce wdyt?